src/quo/components/wallet/progress_bar/view.cljs

```diff
@@ -11,7 +11,7 @@
     [:props
      [:map {:closed false}
       [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
-      [:theme :schema.common/theme]
+      [:theme {:optional true} :schema.common/theme]
```

Comment:
@shivekkhurana, could you help me understand why we need to make this optional? The `view` component always gets the `theme` value injected by `with-theme`. So in which case does the prod code get called without the `theme` key present?

Reply:
When I was writing this, I was assuming that, in case no theme is passed, the default theme is applied.

 So I made it optional. After reading your message, I looked into it and realised that a theme value is expected explicitly. So I'll make this a required field.